scripts/kfctl.sh

```diff
@@ -210,6 +210,7 @@ ksApply () {
   ks apply default -c centraldashboard
   ks apply default -c tf-job-operator
   ks apply default -c argo
+  ks apply default -c katib
```

Comment:
This would mean that katib will get deployed by default into the installations. Is that what we want?

Reply:
I am assuming that 0.3 onwards we want to install katib by default